wordfreq/__init__.py

```diff
@@ -165,17 +84,13 @@ def tokenize(text, lang):
     first, so that they can be expected to match the data.
     """
     if lang == 'ja':
-        global mecab_tokenize
-        if mecab_tokenize is None:
-            from wordfreq.mecab import mecab_tokenize
+        from wordfreq.mecab import mecab_tokenize
```

Comment:
In our use case, we are running each call on large files with several million lines.